### PR TITLE
Soft sync some fixes

### PR DESCRIPTION
**Pull request by Poojawa**
No real fan fair of a CL is needed

### CODE REVIEW

**Comment by Poojawa** [comment]
Oh this assumes we're on TGS4 now

which if we aren't, why the fuck not

File: code/datums/mood_events/needs_events.dm [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	mood_change = -4
 
 /datum/mood_event/wellfed
-	description = "<span class='nicegreen'>My belly feels round and full.</span>\n"
+	description = "<span class='nicegreen'>I'm stuffed!</span>\n"
```
Comment (deathride58) [inline code/datums/mood_events/needs_events.dm:7]: Who's responsible for this

Comment (Poojawa) [reply code/datums/mood_events/needs_events.dm:7]: the mad man who made moods entirely

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm [hunk 2]
```diff
@@ -10,6 +10,8 @@
 	active_power_usage = 100
 	circuit = /obj/item/circuitboard/machine/microwave
 	pass_flags = PASSTABLE
+	light_color = LIGHT_COLOR_YELLOW
+	light_power = 3
```
Comment (deathride58) [inline code/modules/food_and_drinks/kitchen_machinery/microwave.dm:14]: Who's responsible for this atrocious use of light_power

**Review by deathride58 — DISMISSED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm (hunk 2 above)
Comment (Poojawa) [reply code/modules/food_and_drinks/kitchen_machinery/microwave.dm:14]: Moth sympathizers

**Comment by deathride58** [comment]
TGS4 isn't really stable yet, and still has some significant issues. We'll be upgrading to TGS4 as soon as it's relatively stable compared to TGS3.
With the way TGS is made, the current code is backwards compatible with TGS3 due to the presence of TGS3's DMAPI library.

File: code/__DEFINES/tgs.dm [hunk 3]
```diff
@@ -10,6 +10,10 @@
 //Comment this out once you've filled in the below
 #error TGS API unconfigured
 
+//Uncomment this if you wish to allow the game to interact with TGS 3
+//This will raise the minimum required security level of your game to TGS_SECURITY_TRUSTED due to it utilizing call()()
+//#define TGS_V3_API
```
Comment (deathride58) [inline code/__DEFINES/tgs.dm:15]: This is here but it isn't used anywhere in the rest of the diff at all. Did you mirror the full change or just this file?

File: code/modules/food_and_drinks/kitchen_machinery/microwave.dm (hunk 2 above)
Comment (deathride58) [reply code/modules/food_and_drinks/kitchen_machinery/microwave.dm:14]: 🔫

**Review by deathride58 — APPROVED** [review]

**Pull request merged**